ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -778,7 +797,6 @@ class RGBDecomp(object):
         arcpy.env.workspace = indir
         indirbase = os.path.basename(indir)
         # Create a scratch directory for intermediate files
-        scratch = arcpy.CreateFolder_management(indir, "temp")
```

Comment:
`CreateFolder_management` might have side effects that are needed even if the returned `scratch` value isn't used

Reply:
@hjkristenson is it okay to call the function without assignment here